stdlib/posix.pyi

```diff
@@ -59,8 +60,9 @@ F_TEST: int
 F_TLOCK: int
 F_ULOCK: int
 
-GRND_NONBLOCK: int
-GRND_RANDOM: int
+if sys.platform != "darwin":
+    GRND_NONBLOCK: int
+    GRND_RANDOM: int
```

Comment:
These are linux-specific: they are for `getrandom(2)`, and that manual page says it's linux-specific.